shared/helpers/puppeteer.js

```diff
@@ -55,7 +55,10 @@ async function axe(page, overrides = {}) {
 
   // Add preview URL to report violations
   report.violations.forEach((violation) => {
-    violation.helpUrl = `${violation.helpUrl}\n${page.url()}`
+    const { pathname } = new URL(page.url())
+
+    // Replace file:// URLs with Review app
```

Comment:
Quite a complicated string - might be helpful to include an example of what a `violation.helpUrl` looks like?

Reply:
Good shout, added a JSDoc `@example` to show our Review app example URL added on:

```console
You can find more information on this issue here:
https://dequeuniversity.com/rules/axe/4.8/aria-allowed-attr?application=axe-puppeteer
http://localhost:8080/components/radios/with-conditional-items/preview
```